### PR TITLE
Add ApplyEndpoints in helper

### PR DESCRIPTION
**Pull request by xuezhaojun**
Signed-off-by: xuezhaojun <zxue@redhat.com>

### CODE REVIEW

File: pkg/helpers/helpers.go [hunk 1]
```diff
@@ -316,6 +318,30 @@ func ApplyDeployment(
 	return generationStatus, nil
 }
 
+func ApplyEndpoints(ctx context.Context, client coreclientv1.EndpointsGetter, required *corev1.Endpoints) (*corev1.Endpoints, bool, error) {
```
Comment (qiujian16) [inline pkg/helpers/helpers.go:321]: I think it needs a unit test

Comment (xuezhaojun) [reply pkg/helpers/helpers.go:321]: Yes, unit-test added please take another look.

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**